src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
```

Comment:
Quantity nie zawsze musi być integerem. Może być to float w momencie gdyby dochodziło do sprzedaży np. płytek ceramicznych czy podobnego typu produktów, takich gdzie klient określa ilość opakowań. Jako przykład http://www.alekafelki.pl/produkt/negro-lappato-r-60x60

Nie wymuszałbym tutaj typu (mimo że u nas jest sztywny) jeżeli chcesz zostawić pole na modyfikacje.